### PR TITLE
Added initial to .git-authors

### PR DESCRIPTION
**Pull request by linkin-park**
Please review and accept the changes for .git-authors.

### CODE REVIEW

File: .git-authors [hunk 1]
```diff
@@ -8,14 +8,15 @@ pairs:
   nw: Nathan Wang; nathan.wang2
   sc: Serena Cao; wcao0630
   mw: Matt Wills; mattwills8
+  ag: Arun Giridharan; linkin-park
```
Comment (bbmoz) [inline .git-authors]: Hmm what is `linkin-park`? If you are using gmail, this should be the first part

File: .git-authors [hunk 2]
```diff
@@ -8,14 +8,15 @@ pairs:
...
   prefix: pair
   domain: gmail.com
 
 email_addresses:
    kj: karthijey@yahoo.com
    km: klausmana@outlook.com
-   mw: matt_wills8@hotmail.co.uk
\ No newline at end of file
+   mw: matt_wills8@hotmail.co.uk
+   ag: giridharan.arun123
```
Comment (bbmoz) [inline .git-authors]: If you are not using gmail, you should be using your entire email here

File: .git-authors (hunk 1 above)
Comment (linkin-park) [reply .git-authors]: okay :) 
my bad thought need to add username of github after ;

File: .git-authors (hunk 2 above)
Comment (linkin-park) [reply .git-authors]: okay , will get it removed.

**Pull request merged**